### DISCUSSION TITLE
student developer pack

### DISCUSSION THREAD

**Discussion by dodora4011** [community/community#123276] [GitHub Education]
### Hi everyone,

I'm a Korean student and I wrote my information correctly, but the reply told me like this,

Please ensure that your academic affiliation document contains your last name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.

Please ensure that your academic affiliation document contains your first name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). Variations of your name and nicknames are unacceptable.

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.

how can i get it?

**Comment by Brijeshparmarmax** [comment]
I have same problem and it's asking me to verify my email id to school.
